go.mod

```diff
@@ -5,8 +5,9 @@ go 1.16
 require (
 	github.com/go-logr/logr v1.2.2 // indirect
 	github.com/google/uuid v1.2.0 // indirect
-	github.com/onsi/gomega v1.19.0 // indirect
-	github.com/openyurtio/yurt-app-manager-api v0.18.8
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.19.0
+	github.com/openyurtio/yurt-app-manager-api v0.6.0
```

Comment:
how about import `github.com/openyurtio/api`?

Reply:
OK, I change github.com/openyurtio/api

Reply:
Sorry, there's something wrong with the commit message, so I'll start a new PR